dandiapi/api/views/upload.py

```diff
@@ -70,7 +70,7 @@ class UploadCompletionResponseSerializer(serializers.Serializer):
 @swagger_auto_schema(
     method='POST',
     operation_summary='Fetch an existing asset blob by digest, if it exists.',
-    operation_description=f'Supported digest algorithms: {", ".join(supported_digests)}',
+    operation_description=f'Supported digest algorithms: {", ".join(supported_digests.keys())}',
```

Comment:
AFAIK .keys() aren't needed here
```
$> python3 -c 'print(",".join({"1":2, "a":"b"}))'
1,a
```
which allows for a more "generic programming" (e.g. if `supported_digests` where just a tuple or a list of those supported).

```suggestion
    operation_description=f'Supported digest algorithms: {", ".join(supported_digests)}',
```